.travis.yml

```diff
@@ -11,6 +11,8 @@ env:
     - TWILIO_ACCOUNT_SID=ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
```

Comment:
Did you consider removing the env vars from here? Is worthy to note that the only one that is required for the build is `CXX`.